Rakefile

```diff
@@ -6,6 +6,4 @@ require 'rake'
 
 IndyhackersRails::Application.load_tasks
 
-Rake::Task['test'].clear
```

Comment:
I originally removed these lines in my original attempt to figure out why specs were running twice. If someone feels strong enough about having `rake test` run specs, I can see about putting these back.